brainglobe_utils/image_io/load.py

```diff
@@ -41,27 +41,50 @@ def load_any(
     .. warning:: x and y scaling not used at the moment if loading a
         complete image
 
-    :param str src_path: Can be the path of a nifty file, tiff file,
-        tiff files folder or text file containing a list of paths
-    :param float x_scaling_factor: The scaling of the brain along the x
-        dimension (applied on loading before return)
-    :param float y_scaling_factor: The scaling of the brain along the y
-        dimension (applied on loading before return)
-    :param float z_scaling_factor: The scaling of the brain along the z
-        dimension (applied on loading before return)
-    :param bool anti_aliasing: Whether to apply a Gaussian filter to smooth
-        the image prior to down-scaling. It is crucial to filter when
-        down-sampling the image to avoid aliasing artifacts.
-    :param bool load_parallel: Load planes in parallel using multiprocessing
-        for faster data loading
-    :param bool sort_input_file: If set to true and the input is a filepaths
-        file, it will be naturally sorted
-    :param bool as_numpy: Whether to convert the image to a numpy array in
-        memory (rather than a memmap). Only relevant for .nii files.
-    :param bool verbose: Print more information about the process
-    :param int n_free_cpus: Number of cpu cores to leave free.
-    :return: The loaded brain
-    :rtype: np.ndarray
+    Parameters
+    ----------
+    src_path : str
+        Can be the path of a nifty file, tiff file, tiff files folder, or text
+        file containing a list of paths.
+
+    x_scaling_factor : float
```

Comment:
Just a note here: optional arguments take an additional keyword after their type (https://numpydoc.readthedocs.io/en/latest/format.html#parameters).

So `x_scaling_factor : float, optional` as opposed to just `x_scaling_factor : float`. The style guide does say you can write in the default value too, but I personally think that's overkill given it's in the function definition already.

Reply:
Thanks @willGraham01 - I've added the optional tags now. Feel free to merge if you're happy with it :smile: